### DISCUSSION TITLE
Missing Canva Pro Subscription in GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by ankit-nautiyal** [community/community#63555] [GitHub Education]
### Hi everyone,

**I can't  find "Canva Pro" Subscription in GitHub Student Developer Pack, please let me know how can I access it or is it still provided by GitHub ??**

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open.
